src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $main-card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    cursor: pointer;
+  }
```

Comment:
The cursor property should be on the card element itself, not on the hover state. This is to ensure that the cursor indicates a clickable element even before the hover action is performed.